tasklist/client/src/common/components/AssigneeTag/index.tsx

```diff
@@ -15,7 +15,7 @@ import cn from 'classnames';
 
 type Props = {
   currentUser: CurrentUser;
-  assignee: string | null;
+  assignee: string | null | undefined;
```

Comment:
I'm already changing the types here because the v2 APIs return `undefined` instead of just `null`

Reply:
Is the null value still possible?

Reply:
Yes, on V1 user tasks